synapse/storage/schema/__init__.py

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-SCHEMA_VERSION = 65  # remember to update the list below when updating
+SCHEMA_VERSION = 66  # remember to update the list below when updating
```

Comment:
I'm having trouble convincing myself this needs to be bumped? It looks like this is essentially a no-op and should be backwards and forwards compatible?

Or are we just being conservative and bumping it anyway?

Reply:
I believe the issue is that Rich wants to add a `state_key` column to `events`, at which point a rollback becomes impossible because prior versions of Synapse will have these ambiguous queries left in.

Reply:
@clokep did you see my comments at https://github.com/matrix-org/synapse/pull/11497#discussion_r761353811?

Reply:
Ahhhh, yeah I wasn't thinking of rolling forward and then backwards again. 👍

Reply:
> @clokep did you see my comments at https://github.com/matrix-org/synapse/pull/11497#discussion_r761353811?

I have now. :)